### PR TITLE
Add notes from 2018-05-02

### PR DESCRIPTION
**Pull request by jefftrull**
Hi Mike! I'm totally happy to merge with your notes if you like. LMK what you think of these.

### CODE REVIEW

File: meetups/20180502.org [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+#+AUTHOR: with special guest Harry Schwartz
+
+* Tech Check
+*** Wi-Fi
+    :PROPERTIES:
+    :BEAMER_env: frame
+    :END:
+
+- SSID "Triplebyte Guest"
+- Password "welcome01"
```
Comment (wrightmikea) [inline meetups/20180502.org]: shouldn't we redact passwords from publicly visible documents?

File: meetups/20180502.org [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+    :PROPERTIES:
+    :BEAMER_env: frame
+    :END:
+[[file:./triplebyte_logo.png]]
+* Presenter
+*** Harry Schwartz
+    :PROPERTIES:
+    :BEAMER_env: frame
+    :END:
+
```
Comment (wrightmikea) [inline meetups/20180502.org]: I've checked in three scans of the speaker notes that you could update your document to link-to.
look in this directory: https://github.com/Emacs-SF/meetup-notes/tree/master/images/20180502

File: meetups/20180502.org [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+*** Other things that came up
+- there is a macro called defmath you can define your own calc functions with
+- "screenkey" was the tool used to display keys being pressed
+- Harry couldn't find the info files for calc; it turns out they are in a separate package ("emacs25-common-non-dfsg") on Debian (and thus Ubuntu) for license reasons.
+
+** Post-Presentation Discussion
+- We veered off pretty far, into math and category theory
+- Session Idea: how to write an org-mode exporter
+
```
Comment (wrightmikea) [inline meetups/20180502.org]: My other notes (you could add here?):
1. online calc info: 
    > https://www.gnu.org/software/emacs/manual/html_mono/calc.html
2. on Debian, emacs info for calc is missing unless you install it, e.g.:
    $ sudo apt-get install emacs25-common-non-dfsg
3. there was a question about what tool was used to echo keystrokes.  It was screenkey, here's a link: 
    >  https://github.com/wavexx/screenkey
Category theory
Software foundations book (I found the following link, is it correct?):
> https://softwarefoundations.cis.upenn.edu/

and I've been enjoying the book you mentioned in the meetups comments "From Mathematics to Generic Programming" (which I found in my safaribooksonline.com account)
Here's a link to the book on Amazon:
> https://smile.amazon.com/Mathematics-Generic-Programming-Alexander-Stepanov/dp/0321942043/

File: meetups/20180502.org (hunk 1 above)
Comment (jefftrull) [reply meetups/20180502.org]: Oh good one :(

File: meetups/20180502.org (hunk 3 above)
Comment (jefftrull) [reply meetups/20180502.org]: I got screenkey and the info package in there already but not the others :) Thanks!

**Comment by jefftrull** [comment]
OK I think I have everything merged up if you like this :)

**Pull request merged**